.editorconfig

```diff
@@ -38,6 +38,15 @@ generated_code = true
 [CyclomaticComplexityWalker.cs]
 generated_code = true
 
+[**Java.Interop.__TypeRegistrations.cs]
+generated_code = true
+
+[**Uno.UI.UnoRecyclerView.cs]
+generated_code = true
+
+[**Uno.UI.UnoViewGroup.cs]
+generated_code = true
```

Comment:
TODO: Delete. These are not working for a reason I don't know. I added `		<NoWarn>$(NoWarn);CA1825;CA1812</NoWarn>` for now.